ocaml/typing/typecore.ml

```diff
@@ -7992,7 +7998,7 @@ and type_comprehension_expr
         Predef.type_list,
         (fun tcomp -> Texp_list_comprehension tcomp),
         comp,
-        Layout.Type_argument
+        (Layout.Type_argument {parent_path=Predef.path_list;position=1;arity=1})
```

Comment:
Just barely, I think it's worth making a `list_argument_layout` in parallel with the one for option. It would be used here and in predef.